### ISSUE TITLE
Request: TypeORM

### ISSUE THREAD

**Issue by FravonDev** [simple-icons/simple-icons#8147]
### Brand Name

TypeORM

### Website

https://typeorm.io

### Popularity Metric

30.000 stars and 5.6 forks

### Official Resources for Icon and Color

![typeorm-seeklogo com](https://user-images.githubusercontent.com/62142146/208088732-e168fd64-3e48-4f48-b14d-9d91fa7d99f6.svg)

### Contributing

Yes

**Comment by davesagraf** [comment]
@uncenter 

I've made a [PR](https://github.com/simple-icons/simple-icons/pull/10939)

Thanks!

**Comment by davesagraf** [comment]
@uncenter 

Hi again :)

I've noticed some errors in the build here, looked closer and saw some svglint errors.

I started running svglint locally (to avoid commits with failing builds), played around with SVG more at [SVG Viewer](https://www.svgviewer.dev/), scaled & translated the svg content until there were no errors in the svglint. And only then I pushed the latest commit.

Now all the checks have passed :)

Hope this icon makes it into your awesome library.

Thanks!
